src/pages/blog/starlingx-deploying-your-app-1.md

```diff
@@ -0,0 +1,309 @@
+---
+templateKey: blog-post
+title: Deploying your application to StarlingX
+author: Bruno Muniz
+date: 2023-10-13T12:00:00.042Z
+category:
+  - label: Features & Updates
+---
+
+Deploying your own application to StarlingX is as easy as any other Kubernetes
+deployment out there. In this blog post we'll show you the process with a
```

Comment:
```suggestion
deployment out there. In this blog post I will show you the process with a
```

Reply:
As this article has one author, it reads a little confusing to use "we", as it is unclear who that "we" refers to. I would suggest to rather use "I" here and below.

If you are not comfortable with that, you can also use a more active tone and write from the readers perspective. You could say something like: "In this blog post you will learn about the deployment process."

Or, you can stick to a more passive tone, like: "This blog post will go through the deployment process by using a demo application."

Is this helpful?

Reply:
Got it. I guess I was thinking more like "we, the Community, represented here by the person Bruno" or something like that.